tensorflow/tools/ci_build/ci_build.sh

```diff
@@ -35,7 +35,9 @@ fi
 
 # Optional arguments - environment variables. For example:
 # CI_DOCKER_EXTRA_PARAMS='-it --rm' CI_COMMAND_PREFIX='' tensorflow/tools/ci_build/ci_build.sh CPU /bin/bash
-CI_DOCKER_EXTRA_PARAMS=("${CI_DOCKER_EXTRA_PARAMS[@]:---rm}")
+if [[ "${CI_DOCKER_EXTRA_PARAMS}" != *"--rm"* ]]; then
+  CI_DOCKER_EXTRA_PARAMS="--rm ${CI_DOCKER_EXTRA_PARAMS}"
+fi
```

Comment:
Making sure that the "--rm" flag for Docker is always set. In the case of automatic parametric build, CI_DOCKER_EXTRA_PARAMS may already been set to an non-empty value. So the previous approach needs to be replaced.